lib/transport/resource.ex

```diff
@@ -79,6 +81,16 @@ defmodule Transport.Resource do
     |> Repo.update
   end
 
+  def save(%{url: url}, %{"validations" => validations}) when not is_nil(validations) do
```

Comment:
I had a hard time to understand, maybe you can add a comment, on why you do that

Reply:
Actually we should still save it. I changed a bit the logic (and added a comment)